frontend/my-app/src/Pages/LandingPage.tsx

```diff
@@ -69,26 +78,144 @@ const LandingPage: React.FC = () => {
     navigate('/')
   }
 
+  const handleFriendMenu = (): void => {
+    console.log('dummy friend pfp actuation')
+  }
+
   return (
-    <div className="container">
-      <p>Current user: {user.email}</p>
-      <div className="buttons">
-        <button onClick={handleGenerateClick}>Generate</button>
-        <button onClick={handlAccountClick}>Account</button>
-        <button onClick={handleLogoutClick}>Logout</button>
+      <div className="container">
+          <AppBar position="relative"
+          sx={{
+            color: '1a7fc1',
+            alignSelf: 'start',
+            width: 'auto',
+            minWidth: 4 / 5,
+            mb: 'auto',
+            mt: 5,
+            mx: 'auto'
+          }}
+          >
+            <Toolbar>
+              <Button color='secondary' variant='contained' onClick={handleLogoutClick}>Logout</Button>
+              <Stack
+                direction="row"
+                justifyContent="flex-start"
+                alignItems="center"
+                spacing={2}
+                sx={{
+                  display: 'flex',
+                  flexWarp: 'wrap',
+                  m: 'auto'
+                }}
+              >
+                {getCheckedFriends().map((user, index) => (
+                  <IconButton color="primary" aria-label="upload picture" component="label"
+                  sx={{
+                    width: 50,
+                    height: 50,
+                    backgroundColor: 'primary.dark',
+                    '&:hover': {
+                      backgroundColor: 'success.main',
+                      opacity: [0.8, 0.8, 0.8]
```

Comment:
Is there a way to condense these styling aspects to something smaller and resuable?